### PR TITLE
Update to latest core alpha

### PR DESCRIPTION
**Pull request by lailabougria**

### CODE REVIEW

File: src/AcceptanceTests/NServiceBus.DataBus.AzureBlobStorage.AcceptanceTests.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.561" />
+    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.627" />
```
Comment (lailabougria) [inline src/AcceptanceTests/NServiceBus.DataBus.AzureBlobStorage.AcceptanceTests.csproj:17]:
```suggestion
    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.627, 9.0.0)" />
```

File: src/DataBus/NServiceBus.DataBus.AzureBlobStorage.csproj [hunk 2]
```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.561, 9.0.0)" />
+    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.627" />
```
Comment (lailabougria) [inline src/DataBus/NServiceBus.DataBus.AzureBlobStorage.csproj:11]:
```suggestion
    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.627, 9.0.0)" />
```

File: src/Tests/NServiceBus.DataBus.AzureBlobStorage.Tests.csproj [hunk 3]
```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.561" />
+    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.627" />
```
Comment (lailabougria) [inline src/Tests/NServiceBus.DataBus.AzureBlobStorage.Tests.csproj:19]:
```suggestion
    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.627, 9.0.0)" />
```

**Comment by andreasohlund** [comment]
https://github.com/Particular/NServiceBus.DataBus.AzureBlobStorage/blob/master/src/DataBus/NServiceBus.DataBus.AzureBlobStorage.csproj#L11 should be changed as well (not sure its 100% needed but we've always done it this way)

File: src/AcceptanceTests/NServiceBus.DataBus.AzureBlobStorage.AcceptanceTests.csproj [hunk 4]
```diff
@@ -14,7 +14,8 @@
...
```
Comment (lailabougria) [inline src/AcceptanceTests/NServiceBus.DataBus.AzureBlobStorage.AcceptanceTests.csproj:17]:
```suggestion
    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.630" />
```

File: src/AcceptanceTests/NServiceBus.DataBus.AzureBlobStorage.AcceptanceTests.csproj [hunk 5]
```diff
@@ -14,7 +14,8 @@
...
+    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="8.0.0-alpha.627" />
```
Comment (lailabougria) [inline src/AcceptanceTests/NServiceBus.DataBus.AzureBlobStorage.AcceptanceTests.csproj:18]:
```suggestion
    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="8.0.0-alpha.630" />
```

File: src/DataBus/NServiceBus.DataBus.AzureBlobStorage.csproj [hunk 6]
```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.561, 9.0.0)" />
+    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.627, 9.0.0)" />
```
Comment (lailabougria) [inline src/DataBus/NServiceBus.DataBus.AzureBlobStorage.csproj:11]:
```suggestion
    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.630, 9.0.0)" />
```

File: src/Tests/NServiceBus.DataBus.AzureBlobStorage.Tests.csproj [hunk 7]
```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.561" />
+    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.627, 9.0.0)" />
```
Comment (lailabougria) [inline src/Tests/NServiceBus.DataBus.AzureBlobStorage.Tests.csproj:19]:
```suggestion
    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.630, 9.0.0)" />
```

**Review by andreasohlund — APPROVED** [review]

**Pull request merged**